dev/com.ibm.ws.kernel.boot.nested/src/com/ibm/ws/kernel/launch/internal/FrameworkManager.java

```diff
@@ -272,6 +277,30 @@ public Void run() throws Exception {
             }
             // Set the framework variables only if everything succeeded.
             systemBundleCtx = fwk.getBundleContext();
+
+            if (LaunchArguments.isBetaEdition()) {
```

Comment:
We don't need this beta block here.  CheckpointHookFactory services only get used when the beta checkpoint feature is enabled.